examples/src/main/scala/org/apache/spark/examples/ExecutorStartupTest.scala

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
A lot of the other files in the examples folder have that naming scheme though, which is what I based it on: https://github.com/apache/spark/tree/master/examples/src/main/scala/org/apache/spark/examples